.github/workflows/ci.yml

```diff
@@ -24,7 +22,7 @@ jobs:
   release-image:
     runs-on: ubuntu-latest
     needs: tests
-    if: github.event_name == 'push'
+    if: github.event_name == 'push' && github.ref_name == 'master'
     steps:
```

Comment:
These changes should mean it'll run the tests per commit but shouldn't  try to make a release.

Can revert if preferred

Reply:
Looks good to me!

Reply:
wait, actually -- this change is a problem. The `on` conditions in an action are an `OR` between it's children, so it was already running on pushes to `master` OR `pull_requests`, with the if condition ensuring that we only released images on a push to master.

So what this change is actually doing is making it so that we'll *only* run this workflow on pull_requests, and therefore never build images unless we've tagged a release.

Let's revert this file, everything else looks good.